### ISSUE TITLE
SD checkpoints not showing up on plugin and getting 404 error

### ISSUE THREAD

**Issue by ambiSk** [AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin#43]
I am using Mac with Photoshop v24.1.0
I followed the instruction, and setup for a remote connection, but my SD checkpoints are not showing up on the plugin UI.
On pressing the Refresh button, I am getting this log
```
INFO:     127.0.0.1:54185 - "GET /sdapi/v1/sd-models HTTP/1.1" 404 Not Found
INFO:     127.0.0.1:54186 - "GET /sdapi/v1/samplers HTTP/1.1" 404 Not Found
```
No crash happening in the code, just this log

**Comment by AbdullahAlfaraj** [comment]
check if your sd webui server exposes "/sdapi/v1/sd-models"
here is how to do it: 
1. access your sd webui url  in the browser.
2. add "/docs" to the end of the url . e.g ```http://127.0.0.1:7860/``` become ```http://127.0.0.1:7860/docs```
3. check if the "/sdapi/v1/sd-models" endpoint is listed.


 
 
![image](https://user-images.githubusercontent.com/7842232/210357824-036a1c9c-532a-4d43-89c8-b3b2d6832ae4.png)

**Comment by jackhagley** [comment]
Don’t think it’s there!

<img width="1448" alt="Screenshot 2023-01-03 at 13 47 35" src="https://user-images.githubusercontent.com/10913053/210360149-3626325f-67a2-48b5-b824-d729758f565a.png">

**Comment by jackhagley** [comment]
It does have it there - and did have it there - just not with the `--xformers` part. I have added `--xformers` but it makes no difference

**Comment by AbdullahAlfaraj** [comment]
@jackhagley show me a screenshot of your sd webui cmd.
![image](https://user-images.githubusercontent.com/7842232/210383112-489132a5-68a7-45c8-8a87-5ec924459cd2.png)
another thing you might want to try is to update your automatic1111 sd webui to the latest version.

**Comment by ambiSk** [comment]
I have this over in my remote server
just to let you know I setup this for my windows pc and here it works perfectly, but on doing the same on Mac, its give the error
![image](https://user-images.githubusercontent.com/89968892/210522035-078719f5-3cf6-49a9-88c0-148f8c18be06.png)

**Comment by AbdullahAlfaraj** [comment]
@ambiSk thank you for reporting back. I've recently tested a remote setup with colab and I found a bug.

in the setting tab make sure your url doesn't end with "/"
![image](https://user-images.githubusercontent.com/7842232/210606837-3e76e3e3-e121-40c6-8c1d-b9023b5a77e7.png)

this is a bug that I'm going to work on later, but just in case you try a remote setup before I get the change to fix it.
this won't work-> 
```https://09a7351c-d161-4142.gradio.live/```

and this will work -> 
```https://09a7351c-d161-4142.gradio.live```
